ArduCopter/mode_poshold.cpp

```diff
@@ -530,27 +524,30 @@ float ModePosHold::mix_controls(float mix_ratio, float first_control, float seco
 // update_brake_angle_from_velocity - updates the brake_angle based on the vehicle's velocity and brake_gain
 //  brake_angle is slewed with the wpnav.poshold_brake_rate_degs_degs and constrained by the wpnav.poshold_braking_angle_max
 //  velocity is assumed to be in the same direction as lean angle so for pitch you should provide the velocity backwards (i.e. -ve forward velocity)
-void ModePosHold::update_brake_angle_from_velocity(float &brake_angle_cd, float velocity_cms)
+void ModePosHold::update_brake_angle_from_velocity(float &brake_angle_rad, float velocity_ms)
 {
-    float lean_angle;
-    float brake_delta_cd = 100.0f * g.poshold_brake_rate_degs * G_Dt;
+    // slew limit per step (rad)
+    const float brake_delta_rad = radians((float)g.poshold_brake_rate_degs) * G_Dt;
 
-    // calculate velocity-only based lean angle
-    lean_angle = -brake.gain * velocity_cms * (1.0f + 500.0f / (fabsf(velocity_cms) + 60.0f));
+    // velocity-shaped lean angle:
+    //   original: -gain * v_cms * (1 + 500/(|v|+60))
```

Comment:
These comments are useful for review, but I don't think they need to be in the code base long term. Maybe after a while we can come back and remove them.